### PR TITLE
GFORMS-1515 - Add a property to output summary and acknowledgement PD…

### PR DESCRIPTION
**Pull request by DursunGundogan**
…Fs in tabular format

### CODE REVIEW

File: app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala [hunk 1]
```diff
@@ -156,12 +156,16 @@ object AcknowledgementSection {
 
 }
 
-case class AcknowledgementSectionPdf(header: Option[SmartString], footer: Option[SmartString])
+case class SectionPdf(
```
Comment (josef-vlach) [inline app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala:159]: Why was this renamed? `SectionPdf` feels like a strange name, it is hard to guess what does it mean.

Comment (DursunGundogan) [reply app/uk/gov/hmrc/gform/sharedmodel/formtemplate/Section.scala:159]: We should reuse it both case classes `AcknowledgementSection` and `SummarySection`. I'll try to find a better name for it.

**Review by josef-vlach — APPROVED** [review]
